### ISSUE TITLE
Pods do not restart when configmap changes via values

### ISSUE THREAD

**Issue by TomKeyte** [launchdarkly/ld-relay-helm#12]
**Is this a support request?**
No

**Describe the bug**
Pods don't restart when configmap changes via `.Values.relay.environment`. This can be combatted with an annotation such as
```yaml
spec:
  template:
    metadata:
      annotations:
        checksum/config: {{ include ("chart-name.configmap") . | sha256sum }}
```
**To reproduce**
Install the chart, change the environment variables via the `.Values.relay.environment`, and update the deployment. Notice that the pods don't automatically restart to account for the new environment

**Expected behavior**
Pods should restart

**Logs**
If applicable, add any log output related to your problem.

**Helm chart version**
1.0.0

**Additional context**

I'd be happy to make a PR if preffered?

**Comment by cwaldren-ld** [comment]
Hi Tom, we appreciate your report and will look into this. No need to make a PR at this time. 

_Filed internally as 175576_.

**Comment by keelerm84** [comment]
This should be resolved in the latest `1.0.1` version.
